db/migrate/20240320072303_remove_tags_of_type_topic.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveTagsOfTypeTopic < ActiveRecord::Migration[7.1]
+  def change
+    Tag.where(type: "Topic").delete_all
```

Comment:
Might be better to not use the model (e.g. see https://rails.rubystyle.guide/#define-model-class-migrations) but if they are used in other migrations, then maybe it doesn't add any value?

Reply:
I'm extremely risk averse when it comes to db migrations!

- Bit of a nitpick, but `Tag.where(type: Topic.sti_name)` might be better practice I think?

- Why did you pick `delete_all` instead of `destroy_all`?